src/js/components/stock-movement-wizard/outboundImport/subsections/OutboundImportItems.jsx

```diff
@@ -1,16 +1,26 @@
 import React, { useMemo } from 'react';
 
 import PropTypes from 'prop-types';
+import { shallowEqual, useSelector } from 'react-redux';
 
 import DataTable, { TableCell } from 'components/DataTable';
 import DateCell from 'components/DataTable/DateCell';
 import Subsection from 'components/Layout/v2/Subsection';
-import { INVENTORY_ITEM_URL } from 'consts/applicationUrls';
 import useTranslate from 'hooks/useTranslate';
+import { formatProductDisplayName } from 'utils/form-values-utils';
 
 const OutboundImportItems = ({ data, errors }) => {
   const translate = useTranslate();
 
+  const isPalletColumnHasAnyValues = useMemo(() => data.some((it) => it.palletName), data);
+  const isBoxColumnHasAnyValues = useMemo(() => data.some((it) => it.boxName), data);
+
+  const {
+    hasBinLocationSupport,
+  } = useSelector((state) => ({
+    hasBinLocationSupport: state.session.currentLocation.hasBinLocationSupport,
+  }), shallowEqual);
```

Comment:
I was not aware of this second argument before but eslint always highlighted the `useSelector` hook for me and I looked into this.
So apparently the second optional argument is for an _equality function_ that will determine if values have changed and will cause the component to rerender. By default it uses the `===` comparison but react-redux also provides `shallowEqual` that does a better job at this comparison, there is also an option to write your own comparison function or use for example one provided from lodash.

looking at the useSelect that I used I guess it is not needed since the default comparison `===` should be enough to compare `boolean` values so I will change :wink:

Reply:
@drodzewicz wouldn't it be useful for the objects/arrays, but is kinda useless for booleans?
Or if we don't provide the `shallowEqual` (or any other method), it will always be re-rendered?
My point is that the `hasBinLocationSupport` is boolean that is comparable by `===`.

Reply:
@drodzewicz sorry, I have not seen the second paragraph:

> looking at the useSelect that I used I guess it is not needed since the default comparison === should be enough to compare boolean values so I will change 😉